pnpm-lock.yaml

```diff
@@ -15156,6 +15157,33 @@ snapshots:
       '@esbuild/win32-ia32': 0.21.5
       '@esbuild/win32-x64': 0.21.5
 
+  esbuild@0.23.1:
```

Comment:
I'm not a fan that this bump adds a separate `esbuild` package, just because `vite` is a bit late with releases and have their version specified directly. 🙈

Reply:
I agree; it's not a problem; it's just an inefficient result that I nitpick. 😆